### DISCUSSION TITLE
Dropdown menu not entierly visible

### DISCUSSION THREAD

**Discussion by melokki** [tailwindlabs/tailwindcss#2523] [Help] [answered]
Hello :wave: 
I am trying to implement a dropdown menu and I've hit this wall
![my image](https://media.discordapp.net/attachments/683401626190610434/763729828364681216/unknown.png)

I don't have any ideas how to fix this. Does anybody have some ideas on how to make the rest of the dropdown menu appear above the container?

**Comment by melokki** [comment]
what I did is the following, I wrapped the overflow hidden element with a  `<div class="relative"></div>` removed the `relative` class from the button that opens the dropdown and now it seems to be working without breaking the corners. At this moment I am not sure if I broke something else somewhere else. 

Can someone tell me if this is a correct way of fixing this?

**Comment by dhruva81** [comment]
I am also facing same issue. Please help.

**Comment by simonswiss** [comment, marked as answer]
Hey!

This happens because, as you mentioned in [your answer](https://github.com/tailwindlabs/tailwindcss/discussions/2523#discussioncomment-93750), the parent element has a class of `overflow-hidden`.

A common way to prevent this from happening is to have your dropdown menu _outside_ of the element with the `overflow-hidden` class.

So instead of having the dropdown menu inside that element, have it as a sibling element. Since you still want the element to be "`absolute`ly" positioned in relation to the button, you'll need to have another wrapper with the `relative` position (which sounds like what you've done). 

So that your dropdown does not cover your button, you might want to offset that absolutely positioned dropdown from the top with a `top-{n}` class.

If it's not possible in your case, you might need to remove the `overflow-hidden` utility, if that's an option.

Hope it helps!

**Reply by dhruva81** [reply]
Thank you for your tip.
I have removed overflow-hidden and it is visible completely.

Thanks

**Comment by minahpro** [comment]
I always use `opacity-0 invisible` and `opacity-100 visible`  with `transition-all duration-200` to get the best a nicest transition dropdown
read [this](https://medium.com/@minahmmassy/in-2-minutes-how-to-make-a-tailwind-css-dropdown-menu-817320bb0678)
